### ISSUE TITLE
TVShowTime

### ISSUE THREAD

**Issue by henriquev16** [Stremio/stremio-features#21]
Hi! 
I use TV Show Time a lot to keep track of my watched series episodes and I think that it would be nice to have an addon on Strem.io that checks your TV Show Time watched episodes and add that information to Strem.io!

Sorry about my English, I'm Portuguese and I'm trying to improve it! :)

Merry christmas to everyone!

**Comment by Ivshti** [comment]
good point, since we already have an integration for trakt.tv that shouldn't be too hard to add

**Comment by ehammo** [comment]
Any news on this feature?

**Comment by TRtomasz** [comment]
we do not plan to add support for any other trackers at this time
